### DISCUSSION TITLE
Hanging when opening shell folder dialogues on 21354 and newer

### DISCUSSION THREAD

**Discussion by craigloewen-msft** [microsoft/WSL#6801] [General]
Hi folks, you may have seen some hangs when opening up shell folder dialogues on builds 21354 or higher. We believe this could be caused by the fact that we changed the path \\wsl\ to \\wsl.localhost\ by default in WSL (\\wsl$\ will still work and remains unchanged). Your file explorer might still have files pointing to \\wsl\ which could be causing the hang as those won't query to anything. We believe [clearing your recent files](https://windowsloop.com/clear-recent-files-in-file-explorer/#:~:text=Clear%20Recent%20Files%20in%20Windows%2010%201%20First%2C,your%20recent%20files.%20To%20do%20that%2C...%20See%20More.) in file explorer could help resolve this issue. Let us know if you have any other questions, thanks!

**Comment by rlabrecque** [comment]
Thanks for this change, `\\wsl.localhost` seems ideal, and I'm glad to see your team already using it in place of `\\wsl$\` sometimes already!
